### PR TITLE
Trivial editing

### PR DESCRIPTION
**Pull request by bradh**

### CODE REVIEW

File: source/quickinstallationguide/qig.rst [hunk 1]
```diff
@@ -86,10 +86,10 @@ CloudStack. We will go over the steps to prepare now.
 Operating System
 ~~~~~~~~~~~~~~~~
 
-Using the CentOS 7.9.2009 minmal x86_64 install ISO, you'll need to install
+Using the CentOS 7.9.2009 minimal x86_64 install ISO, you'll need to install
 CentOS 7 on your hardware. The defaults will generally be acceptable for this
 installation - but make sure to configure IP address/parameters so that you can later install needed
-packages from internet. Later, we will change the network configuration as needed.
+packages from the internet. Later, we will change the network configuration as needed.
```
Comment (DaanHoogland) [inline source/quickinstallationguide/qig.rst:92]: I am not an English speaker but isn't it either `internet` or `the Internet` ? (note the capitalisation)

File: source/quickinstallationguide/qig.rst [hunk 2]
```diff
@@ -460,14 +460,14 @@ following command:
...
-will automatically install Java 11, but it's good to explicitly confirm that the Java 11 
+will automatically install Java 11, but it's good to explicitly confirm that Java 11 
 is the selected/active one (in case you had a previous Java version already installed):
 
    .. parsed-literal::
 
       $ alternatives --config java
       
-Make sure that Java 11 is the chosen one.
+Make sure that Java 11 is selected.
```
Comment (DaanHoogland) [inline source/quickinstallationguide/qig.rst:470]: I kind of like `the chosen one` ;)

File: source/quickinstallationguide/qig.rst (hunk 1 above)
Comment (DaanHoogland) [reply source/quickinstallationguide/qig.rst:92]: @bradh, so capitalisation is optional if the article is used?

Comment (bradh) [reply source/quickinstallationguide/qig.rst:92]: Nouns (other than pronouns) are not usually capitalised. 

Seriously though, its not a big deal. The only thing I was trying to do in this PR was fix the spelling mistake. I can back out the rest of it.

Comment (DaanHoogland) [reply source/quickinstallationguide/qig.rst:92]: no, don´t bother I'll merge

**Review by DaanHoogland — APPROVED** [review]

**Pull request merged**